src/index.ts

```diff
@@ -43,3 +43,4 @@ export {
   toNanoString,
   mapPaginatedStream,
 } from './utils'
+export { buildDirectMessageTopic } from './utils'
```

Comment:
Should we also do V2 for good measure?

```suggestion
export { buildDirectMessageTopic, buildDirectMessageTopicV2 } from './utils'
```